dynamic_validators/insecure_dynamic_validators.md

```diff
@@ -0,0 +1,103 @@
+# Dynamic Validators (Insecure version)
```

Comment:
The title of this PR and this spec is misleading. It suggests that there are "dynamic validators" which are somehow different from the validators that existed before. I think what you mean is something like:

Spec for adding/removing validators dynamically at runtime